release.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * This script assumes AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY are exported as environment variables
+ */
+const archiver = require('archiver');
+const AWS = require('aws-sdk');
+const fs = require('fs');
+const zipOutput = './archive/blog-posts.zip';
```

Comment:
I know these changes came a bit after this, but's lets match the implementation here, and change the name to _wordpress_
https://github.com/ProvidenceGeeks/website-lambda-meetup/blob/master/release.js#L9
```
const archiveOutputDirectory = './archive';
const s3Bucket = 'lambda.pvdgeeks.org';
const s3Key = `wordpress-${packageJson.version}.zip`;
const zipOutput = `./archive/${s3Key}`;
```

Also note the change to the `release` function
https://github.com/ProvidenceGeeks/website-lambda-meetup/blob/master/release.js#L40